docker/oso-rhel7-zagg-client/root/config.yml

```diff
@@ -67,6 +67,15 @@
     - name: send openshift-master pods count
       hour: "*/2"
       job: ops-runner -n cspodc.openshift.master.pod.count /usr/bin/cron-send-pod-count
+    - name: send cluster user running pod total count
+      hour: "*/2"
+      job: ops-runner -n csurpc.openshift.master.user.pod.running.count /usr/bin/cron-send-user-pod-count "RUNNING"
+    - name: send cluster user pod total count
+      hour: "*/2"
+      job: ops-runner -n csurpc.openshift.master.user.pod.total.count /usr/bin/cron-send-user-pod-count "TOTAL"
```

Comment:
Here there are 2 separate cron jobs. I think there should just be 1.

Something like this:
```
    - name: send cluster user pod total count
      hour: "*/2"
      job: ops-runner -n csurpc.openshift.master.user.pod.count /usr/bin/cron-send-user-pod-count
```